### DISCUSSION TITLE
use in borwser header  error occurred !

### DISCUSSION THREAD

**Discussion by danielmlc** [vuejs/pinia#1490] [Help and Questions] [answered]
### Reproduction

"pinia": "^2.0.17",

### Steps to reproduce the bug

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
  <title> <%= htmlWebpackPlugin.options.title %></title>
  <script src="./lib/pinia/pinia.iife.prod.js"></script>
<body>
  <div id="app"></div>
</body>
</html>

### Expected behavior

pinia has  error occurred !

### Actual behavior

![image](https://user-images.githubusercontent.com/16604995/181415786-94372dfc-6c4f-491f-bc58-95af90d1ae49.png)

**Comment by posva** [comment, marked as answer]
you need to include in this order:

- vue
- vue-demi
- pinia

See https://jsfiddle.net/posva/2ewqyks4/
